static/absolute/js/globals.js

```diff
@@ -0,0 +1,17 @@
+// Get the event ID from the current URL
+function getEventId() {
+  // https://ogene.com/1/participants
+  const currentURL = window.location.href;
+  // https://ogene.com
+  const prefix = window.location.origin;
+  // 1/participants
+  const postfix = currentURL.slice(prefix.length + 1);
+  // 1
+  const slashIdx = postfix.indexOf("/");
+  if (slashIdx !== -1) {
+    return parseInt(postfix.slice(0, postfix.indexOf("/")));
```

Comment:
nit: This is very small, but since you set slashIdx above, why not use it in this slice method instead of calling indexOf again?

Reply:
+1

Reply:
+1